bin/reference_tools/vcf2.pm

```diff
@@ -27,30 +27,30 @@ sub _open {
     my $full_header_str;
 
     while( readline($$self{fh}) ) {
-	chomp;
```

Comment:
There was a mix of tab indentations and space indentations here. That is why there is a diff here

Reply:
Yeah I had some issues with VS Code that messed up a lot of PERL scripts i various repos. Just go for 4 spaces.